ga4gh/datamodel/reads.py

```diff
@@ -217,7 +218,10 @@ def __init__(self, id_, dataFile):
         super(HtslibReadGroup, self).__init__(id_)
         self._setAccessTimes(dataFile)
         self._samFilePath = dataFile
-        self._samFile = pysam.AlignmentFile(dataFile)
+        try:
+            self._samFile = pysam.AlignmentFile(dataFile)
+        except ValueError:
```

Comment:
Not much, just this string: "could not open file (mode='r') - is it SAM/BAM format?"